src/main/scala/rps/Game.scala

```diff
@@ -1,33 +1,55 @@
 package rps
 import scala.io.StdIn.readLine
 
+sealed abstract class Move {
+  def toString : String
+}
+case object ROCK extends Move {
+  override def toString = "Rock"
+}
+case object PAPER extends Move {
+  override def toString = "Paper"
+}
+case object SCISSORS extends Move {
+  override def toString = "Scissors"
+}
+case object INVALID extends Move {
```

Comment:
Let's not have an `Invalid` case.
Instead, when something is invalid, it should be `None`.